aptos-move/framework/aptos-framework/sources/util.move

```diff
@@ -4,7 +4,9 @@ module aptos_framework::util {
     friend aptos_framework::gas_schedule;
 
     /// Native function to deserialize a type T.
-    /// TODO: may want to move it in extra module if needed also in other places inside of the Fx.
-    /// However, should not make this function public outside of the Fx.
-    public(friend) native fun from_bytes<T: copy + drop>(bytes: vector<u8>): T;
+    ///
```

Comment:
this probably needs to move to aptos-stdlib even if it ... requires friends

Reply:
No it is need here, both for code and gas

Reply:
Hmm weird to have it in both framework and aptos-stdlib. Can't we make the framework package depend on the aptos-stdlib and stdlib packages and have just one single from_bytes function in aptos-stdlib?

Reply:
We can't. We don't want to make `from_bytes` public, and a friend can't be used across packages. This is also documented in the code.

Reply:
Hmm I see. Thanks for the clarification.